app/controllers/claims/support/schools/users_controller.rb

```diff
@@ -0,0 +1,46 @@
+class Claims::Support::Schools::UsersController < Claims::Support::ApplicationController
```

Comment:
Does there need to be a 'support'-specific controller for this?

Non-support users within a school can also manage the user list (e.g. they can invite new users, remove users, etc. from their school). So every type of user will be able to manage 'school users'. Won't this lead to duplication if we create one specifically for 'support' users?

e.g. I'm wondering if this should actually be `Claims::Schools::UsersController`

Reply:
My experience has typically led me to separate concerns between user-facing actions and admin-facing actions.

Yes, both can add users, but the side-effects and constraints between the two may differ, especially at the controller level, for example, we may want to send out notifications to admins when a user performs an action or we might want to notify users when an admin performs an invite on their behalf for example. To DRY up the code in terms of functionality, we can always call a shared service object.